flake8_graphql.py

```diff
@@ -17,9 +17,23 @@ class GraphQLChecker(object):
     def __init__(self, tree, *args, **kwargs):
         self.tree = tree
 
+    @classmethod
+    def add_options(cls, parser):
+        parser.add_option(
+            '--gql-identifier',
+            type='string',
```

Comment:
the identifier should be callable, but I dont think this can be enforced from here 🤔 Do you have something particular in mind?